notebooks/ART_tutorial.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
I think we should just copy the latest notebook from the private repo. What do you think?

Reply:
Yes, agreed! I did this work before PR#249, and happy to update this PR now that that one is reviewed/merged.  As-is, this would have copied the version from master prior to that PR, so not been harmful.